src/Faithlife.Utility/StringUtility.cs

```diff
@@ -78,6 +78,38 @@ public static string ReplaceOrdinal(this string source, string oldValue, string
 #endif
 		}
 
+#if NETSTANDARD2_0
+		/// <summary>
+		/// Calls string.Contains with StringComparison.Ordinal.
+		/// </summary>
+#else
+		/// <summary>
+		/// Calls <see cref="string.Contains(char, StringComparison)">string.Contains</see> with <see cref="StringComparison.Ordinal">StringComparison.Ordinal</see>.
+		/// </summary>
+#endif
+		public static bool ContainsOrdinal(this string source, char value) =>
+#if NETSTANDARD2_0
+			source.Contains(value);
```

Comment:
This calls `Enumerable.Contains<char>(this IEnumerable<char> source, char value);`, which is likely not an efficient way to implement this functionality.

```suggestion
			source.IndexOf(value) != -1
```